src/chains/ethereum/ethereum/src/helpers/trie.ts

```diff
@@ -21,7 +21,7 @@ export class GanacheTrie extends SecureTrie {
    */
   copy(includeCheckpoints = true) {
     const db = this.db.copy();
-    const secureTrie = new GanacheTrie(db._leveldb, this.root, this.blockchain);
+    const secureTrie = new GanacheTrie(db._leveldb as LevelUp, this.root, this.blockchain);
```

Comment:
`merkle-patricia-tree` uses a different version of the `CheckpointDB` types so we have to cast it to our own.